### DISCUSSION TITLE
Information for Cloud service provider regions and zones

### DISCUSSION THREAD

**Discussion by seokho-son** [cloud-barista/cb-tumblebug#452] [Show and tell]
## Alibaba
https://www.alibabacloud.com/help/doc-detail/89155.htm
https://www.alibabacloud.com/ko/global-locations?spm=a2c64.11124421.7854525720.1.3436296820rDZI#J_5392902970

**Comment by jihoon-seo** [comment]
## AWS EC2
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html

**Comment by jihoon-seo** [comment]
## Azure Virtual Machines
https://azure.microsoft.com/en-us/global-infrastructure/services/?products=virtual-machines

**Reply by seokho-son** [reply]
https://infrastructuremap.microsoft.com/explore

**Comment by jihoon-seo** [comment]
## GCP Compute Engine
https://cloud.google.com/compute/docs/regions-zones

**Comment by seokho-son** [comment]
## Tencent
https://intl.cloud.tencent.com/document/product/213/6091

**Comment by seokho-son** [comment]
## IBM  vpc
https://cloud.ibm.com/docs/overview?topic=overview-locations

**Comment by jihoon-seo** [comment]
## NHN Cloud
- ko: https://docs.toast.com/ko/TOAST/ko/region-guide/
- en: https://docs.toast.com/en/TOAST/en/region-guide/

**Comment by jihoon-seo** [comment]
## NCP
https://www.ncloud.com/product/global/globalRegion#:~:text=준비%20중에%20있습니다.)-,글로벌%20리전별%20제공%20서비스,-서비스
(여러 번 클릭해야 할 수도 있음)

**Comment by jihoon-seo** [comment]
## KT Cloud
https://cloud.kt.com/portal/user-guide/education-multizone-intro
